packages/x-charts/src/ChartsTooltip/ChartsTooltipTable.tsx

```diff
@@ -1,12 +1,16 @@
-import Box from '@mui/material/Box';
-import Paper from '@mui/material/Paper';
+import Box from '@mui/system/Box';
 import { styled } from '@mui/material/styles';
 import { tooltipClasses } from './tooltipClasses';
 
-export const ChartsTooltipPaper = styled(Paper, {
+export const ChartsTooltipPaper = styled('div', {
   name: 'MuiChartsTooltip',
-  slot: 'Table',
-})({});
+  slot: 'Container',
+})(({ theme }) => ({
+  backgroundColor: (theme.vars || theme).palette.background.paper,
+  color: (theme.vars || theme).palette.text.primary,
+  transition: theme.transitions.create('box-shadow'),
+  border: `1px solid ${(theme.vars || theme).palette.divider}`,
```

Comment:
Looks like the `Paper` might have also had a `borderRadius`.
Are we fine with ditching it by default? 🤔

Reply:
I was waiting for the Argos to check the diff. But it reminds me the tooltip is not tested by Argos since no demo has it open 🤔

Reply:
Yup, this discovery also raised a bulb in my head. 💡 
Could we have a demo with tooltip open on render? 🤔

Reply:
For now, I don't think so, because the open/close state is not controlled

For Argos purpose, we could add a demo latter about tooltip customization which will lead to a screenshot of the tooltip